website/src/controllers/api-v3/members.js

```diff
@@ -231,4 +241,76 @@ api.getChallengeMemberProgress = {
   },
 };
 
+async function sendMessage (sender, userToReceiveMessage, messageData) {
```

Comment:
this function ideally should be in a model, since we don't have an `inbox` model we could put it into the user one maybe?